docs/detections/alerts-view-details.asciidoc

```diff
@@ -27,7 +27,9 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Alert-rendered reason statement*: Provides alert details in a logically-arranged format. Shows high-level details, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Fields are interactive, hover over one to access the available actions.
+* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.
```

Comment:
Nit: I think the full name would be best here. "MITRE" is the name of the organization that does the research, and "ATT&CK" is the name of their knowledge-base.
```suggestion
* *MITRE ATT&CK*: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and subtechniques.
```

Reply:
Makes sense, I'll add this change - thanks!